keyboards/tada68/keymaps/tokyovigilante/rules.mk

```diff
@@ -0,0 +1,16 @@
+MCU = atmega32u4
+BOOTLOADER = qmk-dfu
+FIRMWARE_FORMAT = hex
+
+BOOTMAGIC_ENABLE = no	# Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE = yes	# Mouse keys(+4700)
+EXTRAKEY_ENABLE = yes	# Audio control and System control(+450)
+CONSOLE_ENABLE = no 	# Console for debug(+400)
+COMMAND_ENABLE = yes    # Commands for debug and configuration
+NKRO_ENABLE = yes		# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+RGBLIGHT_ENABLE = no   # Enable keyboard underlight functionality (+4870)
+BACKLIGHT_ENABLE = yes  # Enable keyboard backlight functionality (+1150)
+MIDI_ENABLE = no 		# MIDI controls
+AUDIO_ENABLE = no
+UNICODE_ENABLE = no 		# Unicode
+BLUETOOTH_ENABLE = no # Enable Bluetooth with the Adafruit EZ-Key HID
```

Comment:
You don't need all of this - it can be condensed down to:
```make
BOOTLOADER = qmk-dfu
FIRMWARE_FORMAT = hex
```
I plan to add some "legacy" support for the `lufa-ms` bootloader which will set `FIRMWARE_FORMAT = bin`, so when that happens you'd only need to override `BOOTLOADER` here.

Reply:
Noted, thanks. So this will all be merged with the top-level keyboard config?

Reply:
Yep, and keymap level stuff overrides keyboard level.